tests/UniswapAnchoredViewTest.js

```diff
@@ -0,0 +1,158 @@
+const { encode, sign } = require('../sdk/javascript/.tsbuilt/reporter');
+const { uint, keccak256, time, numToHex, address } = require('./Helpers');
+
+async function setupTokenPairs() {
```

Comment:
Maybe, let's delete these token pairs or simplify their data from here and just use mockedVersion. And then real-world data will be in a second file. 

Or can clean up this thing later.